### PR TITLE
@ import all scss in lib/components

### PR DESCRIPTION
**Pull request by samwinslow**
## Changes

Contributes to #4524. Hotfix to ensure component-level SCSS files are loaded. A more stable fix is needed.

## Checklist

- [ ] All querysets/queries filter by Organization, by Team, and by User
- [ ] Django backend tests
- [ ] Jest frontend tests
- [ ] Cypress end-to-end tests
- [ ] Migrations are safe to run at scale (e.g. PostHog Cloud) – present proof if not obvious

### CODE REVIEW

File: frontend/src/lib/components/SelectBox.tsx [hunk 1]
```diff
@@ -5,7 +5,7 @@ import { List } from 'antd'
 import { DownOutlined, RightOutlined } from '@ant-design/icons'
 import { ActionType, CohortType } from '~/types'
 import { selectBoxLogic } from 'lib/logic/selectBoxLogic'
-import './SelectBox.scss'
```
Comment (timgl) [inline frontend/src/lib/components/SelectBox.tsx:8]: does this mean we're double importing stuff?

Comment (samwinslow) [reply frontend/src/lib/components/SelectBox.tsx:8]: Yes, potentially — I can comment this out.

File: frontend/src/lib/components/SelectBox.tsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { List } from 'antd'
...
+// import './SelectBox.scss' // https://github.com/PostHog/posthog/issues/4524
```
Comment (macobo) [inline frontend/src/lib/components/SelectBox.tsx:8]: It might be needed to keep this around for shared dashboards, not sure.

**Comment by macobo** [comment]
There are other scss imports - why not those?

File: frontend/src/lib/components/SelectBox.tsx (hunk 2 above)
Comment (samwinslow) [reply frontend/src/lib/components/SelectBox.tsx:8]: Avoiding importing the SCSS twice (once in global.scss) — which may or may not cause problems.

**Comment by samwinslow** [comment]
> There are other scss imports - why not those?

Oops, just did a shallow search in lib/components... can go deeper there. Should this be *every* single SCSS file besides `global`? `frontend/src/scenes/insights/Insights.scss`, for example, loads fine.

**Comment by macobo** [comment]
Not sure, seems like second order imports have been affected only. That said, whatever works to work around the current set of issues would be great

**Pull request merged**